packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -211,6 +212,8 @@ class _SnapshotViewState extends State<SnapshotView> with AutoDisposeMixin {
               title: 'Snapshot',
               actionText: 'Analyze Snapshot',
               onAction: controller.loadFakeTree,
+              fileToBeImported:
+                  '$current/lib/src/code_size/stub_data/app_size.dart',
```

Comment:
this seems like a surprising file to be importing... Comment, remove, or add a TODO

Reply:
Added a TODO